### ISSUE TITLE
SC 1.4.13 needs an AT exception (think WCAG2ICT also); also some other scoping issues

### ISSUE THREAD

**Issue by peterkorn** [w3c/wcag21#760]
Some AT make use of "dwell selection" such as Dasher and various soft keyboards.  As such, this SC needs an AT exception, or otherwise needs to take such things into account.  This becomes that much more important in the non-web ICT context.

Further, we are seeing increasing use of voice interfaces, perhaps soon with the growth of computer vision we'll see gesture interfaces.  The tight focus on "receiving and removing pointer hover" doesn't seem like it will stand the test of time.  Can this concept be generalized, with pointer hover moving to a Technique?

**Comment by awkawk** [comment]
Thank you for commenting. For more information about how the AG WG will process this issue, see the following:
- [Issue Processing](https://www.w3.org/WAI/WCAG21/comments/#issueprocessing) - How the WG will discuss the issue
- [Issue Resolution](https://www.w3.org/WAI/WCAG21/comments/#issueresolution) - Identifying and responding to the official WG response
- [Issue Notification](https://www.w3.org/WAI/WCAG21/comments/#issuenotification) - How to limit notifications from GitHub

**Comment by awkawk** [comment]
(Proposed response)
Thank you for the comment.

There is an exception for user agents, and this includes AT (we have noticed that we didn't link to the User Agent glossary item and have made that correction in the editor's draft).

For the part of your comment regarding gesture interfaces, the Working Group has made this SC tightly focused because we cannot be aware of what the future of voice and gesture interfaces will bring, and we will be able to augment or modify the requirements to address these new challenges once we have more clarity on them.

**Comment by DavidMacDonald** [comment]
There is an exception for user agents, and this includes AT (we have noticed that we didn't link to the User Agent glossary item and have made that correction in the editor's draft).

For the part of your comment regarding gesture interfaces, the Working Group has made this SC tightly focused because we cannot be aware of what the future of voice and gesture interfaces will bring, and we will be able to augment or modify the requirements to address these new challenges once we have more clarity on them. We used the term "pointer" instead of "mouse", which leverages the term introduced in the "Pointer Events" spec, as an attempt to incorporate wider the action.

**Comment by awkawk** [comment]
The WG decided on the above response, so we changed the text in the comment containing the proposed response to read "[Official WG Response]". Please confirm is you are satisfied with the response within 1 week. If we haven't heard a response in a week we will regard the resolution as satisfactory.
